Bluewire.Conventions/BranchSemantics.cs

```diff
@@ -30,5 +31,25 @@ public string[] GetRemoteBranchFilters(params BranchType[] types)
         {
             return types.Distinct().Select(t => t.BranchFilter).Where(b => !String.IsNullOrWhiteSpace(b)).Select(b => $"*/{b}").ToArray();
         }
+
+
+        // Assumes we always create a tag when we start a new version
+        public Ref GetVersionZeroRef(SemanticVersion semVer)
+        {
+            return new Ref(semVer.Major + "." + semVer.Minor);
+        }
+
+        // Assumes sementics defined in BranchType.cs
+        public Ref GetVersionLatestRef(SemanticVersion semVer)
+        {
+            switch (semVer.SemanticTag)
+            {
+                case "beta": return new Ref("master");

```

Comment:
```
$ git describe --tags --long --match 17.04 origin/candidate/17.04^^^^^
17.04-414-gbda0964ccc
$ git describe --tags --long --match 17.04 maint/17.05~37
17.04-414-g86ea890c91
```
The second is '17.04.414-beta' if you use `master`. But it's a child of `maint/17.04` and therefore it's an invalid result.